ci-operator/step-registry/ipi/conf/capability/additionalcaps/ipi-conf-capability-additionalcaps-commands.sh

```diff
@@ -134,6 +135,10 @@ while [[ -z "${selected_capability}" ]]; do
         for key in "${!dependency_caps[@]}"; do
             if [[ "${selected_capability}" == "${dependency_caps[$key]}" ]]; then
                 echo "capability ${key} depends on Capability ${dependency_caps[$key]}, so disable ${key} along with ${dependency_caps[$key]}"
+                if (( ocp_minor_version < 19 )) && [[ "${key}" == "Ingress" ]]; then
+                    # skip enabling on 4.18 and previouse version, dependency for Ingress is only applicable on 4.19+.
+                    continue
+                fi
```

Comment:
suggest to move the line 137 behind this if block.

Reply:
updated.